docs/DeveloperGuide.adoc

```diff
@@ -14,6 +14,18 @@ endif::[]
 
 By: `Team T09-B3`      Since: `Feb 2018`      Licence: `MIT`
 
+== Introduction
+
+ProgressChecker is for *students* who *prefer to use a desktop app* to keep track of their learning progress throughout the certain module. (Current version is available for CS2103/T) +
+
+For the current version, students can add their teammates into the app. They can create a new task list through google task. By default, app will display all the link:DeveloperGuide.adoc#Learning-Outcomes[Learning Outcomes] for this week in the task list. Students can use this task list to track their weekly homework and the progress of the project.
```

Comment:
Rephrasing to "add teammate records to their contact list in ProgressChecker", might be clearer?